drf_braces/tests/fields/test_modified.py

```diff
@@ -31,6 +31,18 @@ def test_quantize(self):
         self.assertEqual(field.quantize(Decimal('5.1234567')), Decimal('5.123'))
 
 
+class TestCurrencyField(unittest.TestCase):
+    def test_init(self):
+        field = CurrencyField()
+        self.assertIsNone(field.max_digits)
+
+    def test_to_internal_value(self):
+        field = CurrencyField()
+        self.assertEqual(field.to_internal_value(5.2345), 5.23)
```

Comment:
if you pass `Decimal` input to `to_internal_value`, it will return you `Decimal` back so you can safely compare 2 decimals in the assert